src/components/Reports/GoalCalculator/CalculatorSettings/Categories/SettingsCategory/SettingsCategory.test.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SnackbarProvider } from 'notistack';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import theme from 'src/theme';
+import { GoalCalculationQuery } from '../../../Shared/GoalCalculation.generated';
+import { UpdateGoalCalculationMutation } from './GoalCalculation.generated';
+import { SettingsCategory } from './SettingsCategory';
+
+const router = {
+  query: {
+    accountListId: 'test-account-list-id',
+    goalCalculationId: 'test-goal-calculation-id',
+  },
+};
+const TestComponent = () => (
+  <TestRouter router={router}>
```

Comment:
Can we simplify these providers and use `<GoalCalculatorTestWrapper>`? If you need the goal to have a name, you can update the `GoalCalculatorTestWrapper` mocks.